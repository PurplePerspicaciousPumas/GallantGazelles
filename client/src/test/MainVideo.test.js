import React from 'react';
import MainVideo from '../components/MainVideo.jsx';
import renderer from 'react-test-renderer';

it('renders correctly', () => {
  const tree = renderer.create(
    <MainVideo />
  ).toJSON();
  expect(tree).toMatchSnapshot();
});