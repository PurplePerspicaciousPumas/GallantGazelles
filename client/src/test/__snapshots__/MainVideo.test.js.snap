// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<section>
  <div>
    PREV
  </div>
  <div>
    <section>
      <div>
        <iframe
          allowfullscreen={true}
          frameborder="0"
          height="360"
          mozallowfullscreen={true}
          src="https://player.vimeo.com/video/206456987"
          webkitallowfullscreen={true}
          width="640"
        />
      </div>
    </section>
    <section>
      <div>
        <p>
          Shift
        </p>
        <p>
          Switch between Gmail accounts like a boss
        </p>
      </div>
    </section>
  </div>
  <div>
    NEXT
  </div>
</section>
`;
